doc/nrf/libraries/modem/location.rst

```diff
@@ -32,7 +32,7 @@ The supported location methods are as follows:
 
 * GNSS positioning
 
-  * Uses :ref:`GNSS interface<gnss_interface>` for getting the location.
+  * Uses :ref:`GNSS interface <gnss_interface>` for getting the location.
```

Comment:
Just a comment. Links like this work also without the empty space before the arrow bracket start.